src/features/events/EventCard.tsx

```diff
@@ -6,15 +6,17 @@ export type CFTEvent = {
 }
 
 const EventCard = ({ event }: { event: CFTEvent }) => {
+	const title: string = event.title.toUpperCase();
+
 	return (
-		<article className='bg-cft-light-gray p-6 text-cft-black'>
-			<h1>{event.title}</h1>
-			<h2>{event.date}</h2>
-			<h2>{event.location}</h2>
+		<article className='bg-[#133A4E] text-white p-4 shadow-lg shadow-red-900'>
+			<div className='text-3xl'>{title}</div>
```

Comment:
lets make this an h2 tag for screen readers. I'll end up adding an h1 'events' title on the page for same reason.

Reply:
Done.